api/v1/openlibertyapplication_types.go

```diff
@@ -432,6 +435,8 @@ type OpenLibertyApplicationStatus struct {
 	References common.StatusReferences `json:"references,omitempty"`
 
 	SemeruCompiler *SemeruCompilerStatus `json:"semeruCompiler,omitempty"`
+
+	ObservedGeneration *int32 `json:"observedGeneration,omitempty"`
```

Comment:
Please include a description for the field

Reply:
I added a description and changed the variable type to int64 to match .metadata.generation.